### PR TITLE
[PLAYER-2120] Fix for quality menu getting keyboard focus sometimes when using mouse

### PR DESCRIPTION
**Pull request by jpfranco**
Also took the opportunity to fix the gray shadow on the un-mute button.

### CODE REVIEW

**Comment by aeng7** [comment]
@jpfranco can we add a unit test for this? LGTM otherwise

**Comment by jpfranco** [comment]
@aeng7 sure, added.

File: js/components/controlBar.js [hunk 1]
```diff
@@ -202,7 +205,6 @@ var ControlBar = React.createClass({
           typeof this.qualityBtnElement.focus === 'function') {
         this.qualityBtnElement.focus();
       }
-      this.qualityMenuOpenedWithKeyboard = false;
```
Comment (pilievOoyala) [inline js/components/controlBar.js]: Shouldn't we keep this here ?

**Comment by aeng7** [comment]
LGTM

File: js/components/controlBar.js (hunk 1 above)
Comment (jpfranco) [reply js/components/controlBar.js]: Nope, this was moved to `toggleQualityPopover` which is called below.

**Comment by pilievOoyala** [comment]
LGTM

**Pull request merged**